### PR TITLE
feat(packages): upgrade dependencies

### PR DESCRIPTION
**Pull request by bankyadam**
BREAKING CHANGE

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,46 +3,43 @@
...
   "homepage": "https://github.com/emartech/boar-server",
   "devDependencies": {
-    "chai": "3.5.0",
-    "co-mocha": "1.1.3",
-    "mocha": "3.0.2",
-    "nsp": "2.6.1",
-    "semantic-release": "4.3.5",
-    "sinon": "1.17.5",
-    "sinon-chai": "2.12.0"
+    "audit-filter": "^0.5.0",
```
Comment (BalazsAnkes) [inline package.json]: Please remove the `^` from version

**Review by BalazsAnkes — COMMENTED** [review]
Would be great to apply that one minor finding, other than that, it looks good

**Comment by bankyadam** [comment]
> Would be great to apply that one minor finding, other than that, it looks good

Fixed, please review. Also added the `.npmrc`.

**Review by BalazsAnkes — APPROVED** [review]
LGTM

**Pull request closed without merging**